apps/juicebox_web/assets/js/search/search.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { receiveTerm } from './actions';
+import { addVideo } from '../videos/actions';
+import SearchBar from './search-bar';
+import ResultList from './result-list';
+
+class Search extends Component {
+  constructor(props) {
+    super(props);
+    this.handleVideoAdded = this.handleVideoAdded.bind(this);
+  }
+
+  handleVideoAdded(video) {
+    console.log(this.props.streamId, video);
```

Comment:
Unexpected console statement.

[Read more about it here.](http://eslint.org/docs/rules/no-console)